### ISSUE TITLE
What abilities do you most wish to add to Jarvis?

### ISSUE THREAD

**Issue by fiatrete** [fiatrete/OpenDAN-Personal-AI-OS#9]
Jarvis is the built-in AI Agent in OpenDAN and serves as the primary operational channel for users to access various AI capabilities.  Today, we have released the Jarvis Discord bot. With it, you will be able to communicate with Jarvis on the Discord app on any device and utilize various AI capabilities on the server through Jarvis.

Right now we have developed Stable Diffusion functional module, twitter functional module, Youtube video analysis function module and it's not complicated to add more functional modules with the OpenDAN develop framework.

What abilities do you most wish to add to Jarvis? 
Feel free to let us know.

**Comment by sahildeshpande** [comment]
Any way to use it with an LLM ui like Oobabooga? Will also be nice to add custom personality to Jarvis.

**Comment by fiatrete** [comment]
> Any way to use it with an LLM ui like Oobabooga? Will also be nice to add custom personality to Jarvis.

Yes, using ChatGPT as the LLM engine is just a temporary solution. As a personal AI OS that prioritizes privacy as its key feature, the internal AI agents are certainly driven by locally running LLMs. However, the current level of intelligence in open-source LLMs is not yet sufficient to support stable task execution.

However, in scenarios like companion robots, open-source LLMs can work very well.  [MyGirlGPT](https://github.com/Synthintel0/MyGirlGPT) is our team's another AI agent example project , and it is internally powered by an open-source LLM. Have fun to check it out and play.

**Comment by Synthintel0** [comment]
@fiatrete 
How about smart home control? I'm fed up with these dumb voice assistants we have now.

**Comment by carlcc** [comment]
> @fiatrete How about smart home control? I'm fed up with these dumb voice assistants we have now.

Ha... So coincidence. I've wrote a simple game to mimic that just now.

**Comment by fiatrete** [comment]
> @fiatrete How about smart home control? I'm fed up with these dumb voice assistants we have now.

this feature will be tracking here: https://github.com/fiatrete/OpenDAN-Personal-AI-OS/issues/11
